src/plugins/discover/public/application/main/components/sidebar/discover_field.tsx

```diff
@@ -397,10 +409,50 @@ function DiscoverFieldComponent({
 
   const renderPopover = () => {
     const details = getDetails(field);
+    const dateRange = data?.query?.timefilter.timefilter.getTime();
+    const fieldForStats = multiFields ? multiFields[0].field : field; // TODO: how to handle multifields?
+    const showNewStatsPreviewInDiscover = false; // Toggle this variable to preview new stats locally
```

Comment:
Change to `true` for checking new stats on Discover page 👆
We could also introduce a new Advanced setting, wdyt?